packages/client/components/ScopePhaseArea.tsx

```diff
@@ -70,7 +70,8 @@ const ScopePhaseArea = (props: Props) => {
   const {meeting} = props
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
   const {viewerMeetingMember} = meeting
-  const {user, teamMember} = viewerMeetingMember!
+  if (!viewerMeetingMember) return null
```

Comment:
-1 While I don't have a big issue with breaking the hook rules here, I think it would be better to address the underlying issue. From your issue description it appears as if we should add [`ScopePhaseArea_meeting`](https://github.com/ParabolInc/parabol/blob/e2117f4ba74a893e00925d77275f3dd6c8e19717/packages/client/components/ScopePhaseArea.tsx#L176) to the [`AcceptTeamInvitationPayload`](https://github.com/ParabolInc/parabol/blob/e2117f4ba74a893e00925d77275f3dd6c8e19717/packages/client/mutations/AcceptTeamInvitationMutation.ts#L39) instead

Reply:
I tried adding the `ScopePhaseArea_meeting` fragment to the payload and I also tried adding `viewerMeetingMember` to the payload, but `viewerMeetingMember` could still be null in `ScopePhaseArea`. 

I've pushed an update that uses conditionals to grab the values from `viewerMeetingMember`, so it no longer returns before the hook.